modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage_local/overview-and-comparison-of-local-storage-options
+
+:_mod-docs-content-type: CONCEPT
+[id="comparison-of-solutions-to-provision-node-local-storage_{context}"]
+= Comparison of LVM Storage, LSO, and HPP
+
+[id="comparing-storage-types"]
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files), each anchor ID in a module needs a context at the end of it:
```suggestion
[id="comparing-storage-types_{context}"]
```

Reply:
Added context to the end of all Ids.